server.go

```diff
@@ -1149,6 +1229,11 @@ func (s *Server) handleSSF(span *ssf.SSFSpan, ssfFormat string) {
 		atomic.AddInt64(&metricsStruct.ssfRootSpansReceivedTotal, 1)
 	}
 
+	if !s.IsLocal() {
+		//Increment internal metrics keeping track of protocols received while listening
```

Comment:
Can we move this comment to the Godoc for `incrementListeningProtocol`?

Reply:
Yes we can! Done.